backend/substrapp/compute_tasks/compute_pod.py

```diff
@@ -112,6 +113,7 @@ def create_pod(
         args=None,
         volume_mounts=volume_mounts + gpu_volume_mounts,
         security_context=get_security_context(),
+        resources=get_resources_requirements(cpu_request="1000m", memory_request="1Gi", memory_limit="256Gi"),
```

Comment:
Memory limit seem pretty high to me (not even sure we have this memory available in usual production setups). Based on the [documentation](https://docs.substra.org/en/stable/how-to/deploying-substra/index.html#hardware-requirements), hardware requirements recommend at least 30GB so I suggest:
```suggestion
        resources=get_resources_requirements(cpu_request="1000m", memory_request="1Gi", memory_limit="32Gi"),
```
---

FYI - even though I am not sure the benchmark performed in this case is exhaustive enough: 
<img width="1770" alt="image" src="https://github.com/Substra/substra-backend/assets/135698225/bfa8626b-c419-468d-8d18-10e12c5588e2">

Reply:
I didn't want any task to be OOM killed if the memory was indeed available, that's why I set up a very high limit.
But I can lower it if you think it's an acceptable risk.

Reply:
I mean I think 256 is huge, but we can see how it goes from there. If you don't mind me asking, why having restricted the builder part that much then in comparison?

Reply:
Kaniko pods are unlikely to be scheduled on their own VM because they don't use GPU, so in case of something going wrong with memory usage, it would directly impact all other pods.